R/forcats-package.r

```diff
@@ -0,0 +1,10 @@
+#' forcats.
+#'
+#' Helpers for reordering factor levels (including moving specified levels to
+#' front, ordering by first appearance, reversing, and randomly shuffling), and
+#' tools for modifying factor levels (including collapsing rare levels into
+#' other, anonymising, and manually recoding).
+#'
+#' @name forcats
+#' @docType package
+NULL
```

Comment:
Can you please use `"_PACKAGE"` instead of NULL?  Then you don't need `@docType` and some other stuff should get filled in automatically.